### PR TITLE
Add has_geom_detail attribute to routes/outings lists

### PR DESCRIPTION
**Pull request by asaunier**
Related to https://github.com/c2corg/v6_api/pull/687

### CODE REVIEW

**Review by tsauerwein — APPROVED** [review]
:+1:

**Comment by asaunier** [comment]
Thanks @tsauerwein for reviewing. I'll merge when the API PR is ready to be merged.

File: c2corg_common/fields_outing.py [hunk 1]
```diff
@@ -12,7 +12,6 @@
     'locales.hut_comment',
     'locales.route_description',
     'geometry.geom',
-    'geometry.geom_detail',
```
Comment (asaunier) [inline c2corg_common/fields_outing.py]: Do you really need those changes? If we don't need to add the attribute here, it's simpler to cancel this PR :P

Comment (asaunier) [reply c2corg_common/fields_outing.py]: By the way, the version of v6_common to use (actually the hash of the last commit) is set here in the API: https://github.com/c2corg/v6_api/blob/master/requirements.txt#L51

**Review by julsbreakdown — APPROVED** [review]

**Comment by julsbreakdown** [comment]
merging https://github.com/c2corg/v6_api/pull/687

**Pull request merged**